code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -28,17 +28,37 @@
 /obj/item/ammo_casing/c9mmap
 	desc = "A 9mm bullet casing."
 	caliber = "9mm"
-	projectile_type =/obj/item/projectile/bullet/armourpiercing
+	projectile_type = "/obj/item/projectile/bullet/armourpiercing"
```

Comment:
probably didn't realize you can store actual paths in variables
mainly because that's insane